benchmark/kvbench/test/sequential_custom_traffic_perftest.py

```
@@ -71,37 +114,452 @@ def __init__(
+                raw_xfers = self._storage_backend.get_read_handles(
+                    storage_handle, buffer_chunk, num_handles=num_h
+                )
+            else:
+                raw_xfers = self._storage_backend.get_write_handles(
+                    storage_handle, buffer_chunk, num_handles=num_h
+                )
+            return [
+                StorageXferHandle(xfer, file_path, f"{op_name}_{i}")
+                for i, xfer in enumerate(raw_xfers)
+            ]
+        else:
```

Ruff (0.15.13)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)